_posts/2016-05-17-rubygems-org-2016-push.md

```diff
@@ -0,0 +1,62 @@
+---
+title: RubyGems.org 2016 Push
+layout: post
+author: Nick Quaranto
+author_email: nick@quaran.to
+draft: true
+---
+
+Here's a few ideas for directions to push RubyGems.org into a better place for the rest of 2016. If you love RubyGems and hope to see it improve, this is for you!
+
+<img src="/images/push-heart.gif" width="100%" />
+
+## Team Formation
+
+We've historically had a team of people that have been interested in working on RubyGems.org. When Gemcutter first started, I used to give commit access to anyone who had a Pull Request or commit accepted into the repo, following the pattern that Rubinius used. As time went on, that fell out of practice. People would still get involved through the same channels: GitHub Issues or helping on the #rubygems IRC channel, but it's kind of unclear now how people join.
+
+Recently, [RubyTogether](http://rubytogether.org) has helped out by paying developers to work on RubyGems.org and Bundler. This is huge! But from the outside it's hard to tell how this works, or how to get involved. We have a big [Contribution Guide](http://guides.rubygems.org/contributing/) but it spans across all RubyGems projects. Our [CONTRIBUTING.md](https://github.com/rubygems/rubygems.org/blob/master/CONTRIBUTING.md) covers mostly technical aspects and not the social ones.
+
+Let's make this process open. Here's what I'd love to see:
+
+1. A new team page on RubyGems.org showing who's on it, and what roles are (active, advisory, retired, etc)
+1. Explain how RubyGems.org relates to other projects + organizations in the RubyGems ecosystem better
+1. Clear, defined ways to get on the team, how to "move up" from being on the outside, to committing, to deploying, and how to "retire" if need be
+
+We owe it to the community and to ourselves to make this an empowering and inclusive process, and to keep this critical piece of infrastructure not just well-oiled, but well-kept.
+
+
+## Policies
+
+We're long past due forming real written policies about how RubyGems.org makes decisions and treats our users' data. This is due to the a legacy of being a volunteer run service, but that's a bad excuse for not having any formal ways of handling situations. Thanks to some collaboration with NPM, we have a great start to a suite of policies for our packages and servicing our community. Alongside of our established [Code of Conduct](https://github.com/rubygems/rubygems.org/blob/master/CONDUCT.md), we have draft policies ready for:
+
+1. Gem Name Disputes
+2. Privacy
+3. Receiving Abuse Reports
```

Comment:
This is based directly off of https://github.com/npm/policies/blob/master/receiving-reports.md for now. We still have a lot of work to do on the policies and we're seeking some legal review of them too.